src/Referent/ManagedUsersFilter.php

```diff
@@ -273,4 +306,41 @@ public function includeCitizenProject(): bool
     {
         return $this->includeCP;
     }
+
+    public function serialize()
```

Comment:
typehint ?

Reply:
it is `\Serializable` interface signatures